### PR TITLE
Jfr parse

### PR DESCRIPTION
**Pull request by leogomes**
- Bumped to Java 9;
- Added JMC jars;
- Added config library.

Please note that you will need to run the install-mc-jars.sh script now before using the repository.

### CODE REVIEW

**Comment by leogomes** [comment]
@nbogojevic would be nice if you could test if it works for you on windows :)

File: src/main/resources/reference.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+flamegrapher {
```
Comment (nbogojevic) [inline src/main/resources/reference.conf]: How about JSON or simple properties format rather than HOCON

File: install-mc-jars.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# limitations under the License.
+# ----------------------------------------
+# 2018-02-08 Modified to work with JDK 9
+# ----------------------------------------
+#
+# ----------------------------------------------------------------------------
+# Installing JMC Jars to a local repo
+# ----------------------------------------------------------------------------
+
+jmc_version=`echo $(basename $JAVA_HOME/lib/missioncontrol/plugins/com.oracle.jmc.*.jar) | sed 's/.*_\(.*\)\.jar/\1/'`
```
Comment (nbogojevic) [inline install-mc-jars.sh]: This doesn't work with coreutils neither on ubuntu 16.03 nor on git bash for windows. Could be fixed by adding -a option basename which would probably break it on mac, or by using com.oracle.jmc.common.*.jar as path.

**Review by nbogojevic — CHANGES_REQUESTED** [review]
Doesn't compile with GNU coreutils. With proposed fix works on Windows or Ubuntu.

File: src/main/resources/reference.conf (hunk 1 above)
Comment (leogomes) [reply src/main/resources/reference.conf]: Yes, will go for simple properties to start :)

Comment (leogomes) [reply src/main/resources/reference.conf]: Done. Let me know if all work OK for you now.

File: install-mc-jars.sh (hunk 2 above)
Comment (leogomes) [reply install-mc-jars.sh]: Added -a and it works OK on Mac with JDK 9

**Review by nbogojevic — APPROVED** [review]

**Pull request merged**